### PR TITLE
Joshua s brown apache2 gcs data fed web compose fix

### PR DESCRIPTION
**Pull request by JoshuaSBrown**
# Description

The docker compose build files were becoming cluttered, in this pr the build process is separated into three different folders.

* build all containers and run
* build repo containers and run
* build metadata core containers and run

Master scripts were created to assist with the build and run process they were prepended with compose_ and moved into /scripts this way each of the three compose options could use the same master scripts thus making the code DRY.

### CODE REVIEW

File: repository/docker/Dockerfile.gcs [hunk 1]
```diff
@@ -147,3 +152,4 @@ WORKDIR ${DATAFED_INSTALL_PATH}/authz
 # NOTE If you do not overwrite the entrypoint file it will use the GCS instance
 # provided one
 ENTRYPOINT ["/datafed/source/repository/docker/entrypoint_authz.sh"]
+#ENTRYPOINT ["sleep", "infinity"]
```
Comment (nedvedba) [inline repository/docker/Dockerfile.gcs:155]: Remove comment

File: repository/docker/entrypoint_authz.sh
Comment (nedvedba) [inline repository/docker/entrypoint_authz.sh:1]: Looks to be a lot of debug information printing left in this file

**Comment by JoshuaSBrown** [comment]
@nedvedba I want you to get better about holding me accountable for PRs, I haven't provided a description about what problem it solves, updated the changelog or assigned labels to this PR.

File: repository/docker/entrypoint_authz.sh
Comment (JoshuaSBrown) [reply repository/docker/entrypoint_authz.sh:1]: Good catch

**Pull request merged**